recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "3.1.0" %}
+{% set version = "3.1.0-1" %}
```

Comment:
The only [difference between `3.1.0`...`3.1.0-1`]( https://github.com/cython/cython/compare/3.1.0...3.1.0-1 ) is how the changelog renders on PyPI

Don't think we need to worry about that here